model/acl/acl_enumer.go

```diff
@@ -1,24 +1,25 @@
 // Code generated by "enumer -type=ACL -json -transform=snake"; DO NOT EDIT.
 
+//
 package acl
 
 import (
 	"encoding/json"
 	"fmt"
 )
 
-const _ACLName = "catbulkclustersearchremotecreatecountscriptsdeletedocsourcefield_capscloseanalyzeexistsgettemplateexplainindicesaliasaliasesdelete_by_querycacheindexmappingflushforcemergeupgradesettingsopenrecoverymappingsrolloverrefreshsegmentsshrinksplitshard_storesstatsingestvalidatemsearchmgetnodesmtermvectorsreindexupdate_by_queryrenderrank_evalsearch_shardssnapshottaskstermvectorsupdate"
+const _ACLName = "catbulkclustersearchremotecreatecountscriptsdeletedocsourcefield_capscloseanalyzeexistsgettemplateexplainindicesaliasaliasesdelete_by_querycacheindexmappingflushforcemergeupgradesettingsopenrecoverymappingsrolloverrefreshsegmentsshrinksplitshard_storesstatsingestvalidatemsearchmgetnodesmtermvectorsreindexupdate_by_queryrenderrank_evalsearch_shardssnapshottaskstermvectorsupdatereactive_search"
```

Comment:
@deepjyoti30 An ACL should be exact, here it should be `reactivesearch` instead of `reactive_search`.

In my understanding, we use this primarily for assigning ES ACLs but haven't used it for RS API's ACLs, so that's another question on whether to use this mechanism and if we do, then which ACLs should get classified.

Reply:
@siddharthlatest Made the suggested changes